### PR TITLE
Fix: still apply needed Artifact Registry settings when no credentials are found

### PR DESCRIPTION
**Pull request by yihanzhen**

### CODE REVIEW

**Review by megan-kuo — APPROVED** [review]

File: artifactregistry-gradle-plugin/src/main/java/com/google/cloud/artifactregistry/gradle/plugin/ArtifactRegistryGradlePlugin.java [hunk 1]
```diff
@@ -83,7 +84,7 @@ public void setPassword(String password) {
 
   @Override
   public void apply(Object o) {
-    ArtifactRegistryPasswordCredentials crd;
+    ArtifactRegistryPasswordCredentials crd = null;
```
Comment (Gridasov) [inline artifactregistry-gradle-plugin/src/main/java/com/google/cloud/artifactregistry/gradle/plugin/ArtifactRegistryGradlePlugin.java:87]: By default it is null, there is no need to initialize it with null.

**Review by Gridasov — APPROVED** [review]
LGTM

**Review by danielfbright — APPROVED** [review]

File: artifactregistry-gradle-plugin/src/main/java/com/google/cloud/artifactregistry/gradle/plugin/ArtifactRegistryGradlePlugin.java (hunk 1 above)
Comment (yihanzhen) [reply artifactregistry-gradle-plugin/src/main/java/com/google/cloud/artifactregistry/gradle/plugin/ArtifactRegistryGradlePlugin.java:87]: It does not build without this - java complains that `crd` may have not initialized without this.

**Pull request merged**